deployment/deploy-core-infrastructure.ps1

```diff
@@ -39,7 +43,8 @@ if ($null -eq (Get-Module -ListAvailable -Name powershell-yaml)) {
 }
 
 class Aks {
-  [PSCustomObject] Prepare ([string]$resourceGroupName, [string]$aksName, [PSCustomObject]$proxyConfig, [bool]$enableArc, [string]$arcLocation) {
+  [PSCustomObject] Prepare ([string]$resourceGroupName, [string]$aksName, [PSCustomObject]$parentProxyConfig, 
```

Comment:
maybe we should introduce a new featureConfig object that holds all properties for enabling/disabling features like arc, observability and the names etc. to reduce the parameters for the PowerShell func

Reply:
After offline conversation we have decided to keep as is.